tests/unit/OptionsListUtilsTest.js

```diff
@@ -7,99 +7,113 @@ import CONST from '../../src/CONST';
 import * as Report from '../../src/libs/actions/Report';
 
 describe('OptionsListUtils', () => {
+    let reportCount = 0;
+
     // Given a set of reports with both single participants and multiple participants some pinned and some not
     const REPORTS = {
-        1: {
+        [++reportCount]: {
```

Comment:
necessary?

Reply:
Again, not strictly necessary. I added a unit test [here](https://github.com/Expensify/App/pull/8436/files#diff-44ebcab9bae929692fc02b00f963c7bf517c6fb6093d3a5aef5dbbc0eed30468R106-R116), and when I did that I had to make a lot of changes to the indexes throughout this file. These changes hopefully will make it easier/quicker to add or remove tests from this file.

Reply:
>make a lot of changes to the indexes throughout this file

Which changes and why did you have to make them? It looks like we just added a new report to test? 

I am finding it a bit hard to understand the test now since we are using `index++` or `++index` instead of leaving a concrete number to reference when comparing the assertions to the declared data.